test/unit/codegen/codegen_cpp_visitor.cpp

```diff
@@ -422,3 +422,60 @@ SCENARIO("Check code generation for TABLE statements", "[codegen][array_variable
         }
     }
 }
+
+SCENARIO("Check that BEFORE/AFTER block are well generated", "[codegen][before/after]") {
+    GIVEN("A mod file full of BEFORE/AFTER of all kinds") {
+        std::string const nmodl_text = R"(
+            NEURON {
+                SUFFIX ba1
+            }
+            BEFORE BREAKPOINT {}
```

Comment:
Wondering if we should check how it looks like with something inside the before/after block?

Reply:
I agree with Omar, this would be nice to have